test/unit/operations/get_more.test.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+
+import { Long } from '../../../src/bson';
+import { MongoRuntimeError } from '../../../src/error';
+import { GetMoreOperation } from '../../../src/operations/get_more';
+import { Aspect } from '../../../src/operations/operation';
+import { ReadPreference } from '../../../src/read_preference';
+import { Server } from '../../../src/sdam/server';
+import { ServerDescription } from '../../../src/sdam/server_description';
+import { Topology } from '../../../src/sdam/topology';
+import { ClientSession } from '../../../src/sessions';
+import { MongoDBNamespace } from '../../../src/utils';
+
+describe('GetMoreOperation', function () {
+  const ns = new MongoDBNamespace('db.coll');
+  const cursorId = Object.freeze(Long.fromNumber(1));
+  const options = {
+    batchSize: 100,
+    maxTimeMS: 500,
+    readPreference: ReadPreference.primary
+  };
+
+  describe('#constructor', function () {
```

Comment:
Doesn't have to land in this test file, is there a way we can check that sending falsey comments works since we went out of our way to check only for undefined?

Reply:
Yeah, good catch.

I added an integration test to cover this scenario.  Rather than testing every option, I centralized the logic that applies the comment into a utility function and just tested a single scenario.  If you'd prefer not to have a function call (I think Daria has in the past expressed concerns about using function calls for simple checks like this due to performance considerations), I can write a test for each operation that supports `comment`.

Reply:
Absolute fire test builder 🔥 thank you!